nixos/modules/services/video/wivrn.nix

```
@@ -0,0 +1,122 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.wivrn;
+in
+{
+  options = {
+    services.wivrn = {
+      enable = mkEnableOption (lib.mdDoc "WiVRn, an OpenXR streaming application.");
```

```suggestion
      enable = mkEnableOption "WiVRn, an OpenXR streaming application";
```

`mdDoc` is a no-op, and `mkEnableOption` arg shouldn't have trailing period